src/Resources/config/seo_block.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Sonata\MediaBundle\Block\Breadcrumb\GalleryIndexBreadcrumbBlockService;
+use Sonata\MediaBundle\Block\Breadcrumb\GalleryViewBreadcrumbBlockService;
+use Sonata\MediaBundle\Block\Breadcrumb\MediaViewBreadcrumbBlockService;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ReferenceConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // Use "service" function for creating references to services when dropping support for Symfony 4.4
+    $services = $containerConfigurator->services();
+
+    $services->set('sonata.media.block.breadcrumb_view', GalleryViewBreadcrumbBlockService::class)
+        ->tag('sonata.breadcrumb')
+        ->tag('sonata.block', ['context' => 'breadcrumb'])
+        ->args([
+            new ReferenceConfigurator('twig'),
+            new ReferenceConfigurator('knp_menu.menu_provider'),
+            new ReferenceConfigurator('knp_menu.factory'),
```

Comment:
Here we have the dependency with knp-menu (only when using seo-bundle too).

Reply:
Does this fix need to be Backport to 3.X ?

Reply:
This is kind of a fix, because in the real world, people using sonata most likely will add this bundle already because of admin-bundle requiring it, but IMO it should be explicit on composer.json. So it can be backported to 3.x, but since it is not critical, I would prefer if someone else can do it, I would like to focus on 4.0